octobot_trading/personal_data/orders/order_state.py

```diff
@@ -113,12 +113,13 @@ async def update(self) -> None:
         Necessary when the state is not already synchronizing and when the order type is supported by the exchange
         Try to fix the pending state or terminate
         """
-        if self.is_pending() and not self.is_refreshing() and not self.order.is_self_managed():
-            self.log_order_event_message("synchronizing")
-            await self.synchronize()
-        else:
-            async with self.lock:
-                await self.terminate()
+        if not self.is_refreshing():
```

Comment:
I think we should add an `else` with a log in case we want to refresh a refreshing order to keep track of the fact that we tried